stormpath/src/test/java/com/stormpath/sdk/StormpathConfigurationTest.java

```diff
@@ -0,0 +1,45 @@
+package com.stormpath.sdk;
+
+import org.junit.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class StormpathConfigurationTest {
+
+    @Test(expected = IllegalStateException.class)
+    public void exceptionThrownIfBaseUrlNotSet() throws Exception {
+        new StormpathConfiguration.Builder().build();
+    }
+
+    @Test
+    public void pathsStartWithSlash() throws Exception {
+        StormpathConfiguration stormpath = new StormpathConfiguration.Builder()
+                .baseUrl("https://example.com")
```

Comment:
baseUrl should always end with a version. maybe that should be enforced? I am not sure if we do this in the Java SDK.

`https://api.stormpath.com/v1`